packages/flutter/lib/src/semantics/semantics_event.dart

```diff
@@ -86,11 +86,16 @@ class AnnounceSemanticsEvent extends SemanticsEvent {
   /// This property must not be null.
   final TextDirection textDirection;
 
+  /// Determines whether aria announcement needs to be 'polite' or 'assertive' in Flutter web
+  /// Default mode is 'assertive'. Set to false for 'polite'
+  final bool? assertiveAnnouncement;
```

Comment:
Wondering whether a bool is the right choice for this or whether we should use a more descriptive enum with values `polite` and `assertive`. When you set the boolean to false it is not super-clear that the opposite of that is `polite`. An enum would be self-descriptive here.